pots/api.py

```
@@ -1,80 +1,537 @@
-            pots = Pot.objects.all()
-            results = self.paginate_queryset(pots, request, view=self)
-            serializer = PotSerializer(results, many=True)
-            return self.get_paginated_response(serializer.data)
+                ]
+            ),
+            404: OpenApiResponse(description="Voter not found"),
+            500: OpenApiResponse(description="Error fetching voter data"),
+        },
+    )
+    @method_decorator(cache_page(86400))
+    def get(self, request: Request, voter_id: str, *args, **kwargs):
```

Ruff (0.14.5)

Unused method argument: `request`

(ARG002)